### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brentschneider**
Updating getting started steps to include the build step

## Summary

- Overview of changes
- A link to the Jira ticket for quick reference

## How to test

1. Instructions on how to test the changes in this PR.

## Checklist

- [ ] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title`

### If this is a change to code:

- [ ] Created or updated unit tests to cover the logic added
- [ ] If necessary, updated unit-test snapshots (`yarn test:unit:update`) and browser-test snapshots (`yarn test:browser:update`)

### If this is a change to documentation:

- [x] Checked for spelling and grammatical errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,10 @@ This project uses [Yarn](https://yarnpkg.com/) for package management. Yarn help
 
 1. `yarn install`
    - This will also run [Lerna](https://lerna.js.org/) `bootstrap` which allows us to have multiple packages within the same repo (a monorepo). Lerna installs all our dependencies and symlinks any cross-dependencies.
-1. `yarn start`
+2. `yarn build`
```
Comment (forrestbaer) [inline README.md:36]: I kinda feel like we should just remove this 'getting started' section entirely, the Scripts section right after it details all the scripts.

Comment (brentschneider) [reply README.md:36]: Agreed, with e7504d5 work already underway (which is what prompted this change) `install` and `start` would get folks up and running first time around.

Comment (pwolfert) [reply README.md:36]: We probably need to update the entry for `yarn start`, though, to explain that `yarn build` is a prerequisite step

Comment (pwolfert) [reply README.md:36]: And we could add `yarn install` at the very top and move `yarn start` after `yarn build`

**Review by pwolfert — APPROVED** [review]
LGTM 👍 Thanks!

**Pull request merged**